src/cdk/utils/basic-authorizer-handler/index.ts

```diff
@@ -42,7 +42,11 @@ function createAllowPolicy(
 export const handler: CustomAuthorizerHandler = (event, _context, callback) => {
   if (!process.env.USERNAME) {
     callback(new Error('USERNAME is not defined.'));
-  } else if (isValidBasicAuthHeader(event.headers?.authorization)) {
+  } else if (
+    isValidBasicAuthHeader(
+      event.headers?.authorization ?? event.headers?.Authorization
```

Comment:
Please normalize the value with e.g. toLowerCase and then check it.

Reply:
Extracted case insensitive auth header lookup to a function + added test for this new function.

But Jest requires 100% coverage of the whole file? Is this set in stone?

Reply:
I just see outsourcing is not an option. We're not bundling the file. So the generic one doesn't make sense anymore either. Well then you can exclude the lambda in the Jest Config for coverage.